packages/cozy-client/src/models/file.js

```diff
@@ -1,5 +1,4 @@
-import sub from 'date-fns/sub'
-import add from 'date-fns/add'
+import { add, sub } from 'date-fns'
```

Comment:
We will not import from the root for this kind of lib. Too much stuff can comes with that kind of import.